.github/ISSUE_TEMPLATE/incorrect-syntax-highlighting.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Incorrect syntax highlighting
+about: Report incorrect syntax highlighting...
+title: "([language name]) Short description of issue..."
+labels: language, bug, help welcome
+assignees: ''
+
+---
+
+**Describe the issue**
+A clear and concise description of what the issue seems to be.
+
+**Which language seems to have the issue?**
+Please specify exactly *which* language grammar you are using to highlight (`python`, `javascript`, etc.) or specify if you are using auto-detection.  If you are using auto-detection please first check to make sure the language you are expecting was properly detected.  If Highlight.js misidentifies the type of code - then the highlighting may be entirely incorrect - and there often isn't much we can do about that - auto-detection is only best-effort and can't guess right 100% of the time.
+
+**Sample Code to Reproduce**
+Please include text examples of the code that fails to highlight properly or can reproduce the bug.  You can attach a screenshot if you'd like, but the code is necessary because we may need to copy/paste it to help resolve the issue - or add it to a test.
+
+A jsfiddle can sometimes be even better.  You can fork an example test case:
+https://jsfiddle.net/ajoshguy/gfzujpyt/5/
```

Comment:
Should we set a [base version](https://docs.jsfiddle.net/getting-started#setting-fiddle-as-base-version) here? This way we wouldn't have to update this template each time there's a new highlight.js release.

Reply:
How would we do that exactly?

Reply:
https://docs.jsfiddle.net/getting-started#setting-fiddle-as-base-version

Reply:
Ah yes, for sure.